webhooks-extension/DEVELOPMENT.md

```diff
@@ -46,10 +46,21 @@ data='{
   "accesstoken": "github-secret",
   "pipeline": "simple-pipeline",
   "dockerregistry": "mydockerregistry"
+  "pulltask": "monitor-result-task",
+  "onsuccesscomment": "OK",
+  "onfailurecomment": "ERROR",
 }'
 curl -d "${data}" -H "Content-Type: application/json" -X POST http://localhost:8080/webhooks
 ```
 
+`pulltask`: task name that monitors the pipeline and updates the pull request when it finishes. The default task is `monitor-result-task`
+
+`onsuccesscomment`: comment text that is put into the pull request when the pipelinerun finishes successfully.  The default is `OK: <pipelinerun name>`
+
+`onfailurecomment`: comment text that is put into the pull request when the pipelinerun finishes with errors.  The default is `ERROR: <pipelinerun name>`
+
+The rest of items are mandatory in the POST data. 
```

Comment:
I'm not sure I'm a fan of this ...... I would delete this and have a section before describing these 3 settings (i.e line 55) that states: "Whilst most parameters are mandatory the following three are optional.  Not setting them will allow the defaults to take effect and this is recommended in most cases."

Reply:
OK.  I changed.